src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -36,6 +36,7 @@
 import agent_utils
 import server_utils
 import write_host_file
+import spdk
```

Comment:
(pylint-import-error) Unable to import 'spdk'
(pylint-unused-import) Unused import spdk